.changeset/hip-jokes-press.md

```diff
@@ -1,2 +1,5 @@
 ---
----
\ No newline at end of file
+"@kaizen/components": patch
+---
+
+Dependency updates
```

Comment:
Something like this. This gets added to the changelog, so we would want to know exactly what changed in this release.

```suggestion
Dependency updates.

- "@cultureamp/i18n-react-intl": "^1.6.2"
- "@rollup/plugin-node-resolve": "^15.2.1"
- "@rollup/plugin-typescript": "^11.1.3"
- "sass": "^1.66.1"
```